wallets/rn_cli_wallet/src/components/DiscoverList.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import {FlatList} from 'react-native';
+
+import projectsData from '../constants/projects-response.json';
+import DiscoverListItem from '../components/components/DiscoverListItem';
+import SubscriptionListTabHeader from '@/components/SubscriptionListTabHeader';
+
+type DiscoverListProps = {
+  page: number;
+  setPage: (index: number) => void;
+};
+
+export default function DiscoverList({page, setPage}: DiscoverListProps) {
+  return (
+    <FlatList
+      contentInsetAdjustmentBehavior="automatic"
+      contentContainerStyle={{
+        gap: 16,
+      }}
+      ListHeaderComponent={() => (
+        <SubscriptionListTabHeader page={page} setPage={setPage} />
+      )}
+      data={projectsData}
+      renderItem={({item}) => <DiscoverListItem item={item} />}
```

Comment:
add this performance improvement, where ITEM_HEIGHT is the height of DiscoverListItem. You should set a fixed Height to that component

```
getItemLayout={(_, index) => ({
        length: ITEM_HEIGH,
        offset: ITEM_HEIGH * index,
        index,
      })}
```

Reply:
I don't think we need this since the Discover list is not expected to have thousands of list items. Also discover list item have dynamic height since we don't truncate the description of it so it might be one line or multiple. I'd prefer not to overengineering this. But totally great call to keep in mind while working on lists. 🙏